docs/schema.md

```diff
@@ -854,11 +859,51 @@ A, B, C, D, E, F`. This is the desired behaviour in most cases.
 If `limit` is set to `Y`, the list of affected commits will be `X, A, B, C`. This
 is equivalent to `git rev-list X..Y` (but including `X` and excluding `Y`).
 This may be useful if the scope of a vulnerability entry is limited to a small
-set of linear branches. Multiple `"limit"` events may be specified for each
+set of linear branches. Multiple `limit` events may be specified for each
 branch -- each expands the scope of the git commit graph to cover.
 
 Note that we did not specify a `fixed` event here as `limit` makes it redundant.
 
+#### `last_affected` vs `fixed` example
+The difference between `last_affected` and `fixed` is subtle and these examples will 
+clarify the use of these fields. 
+
+The following example expresses that the vulnerability is present in all versions
+of the package, up to and including version `2.1.214`. Versions above `2.1.214` are
+assumed to be free from the vulnerability, but there is a potential for a false
+negative. The `last_affected` field is typically assigned at the time of discovery and 
+assumes the vulnerability will be addressed in the following version. 
+
+
+```json
+"ranges":[ {
+  "type":"ECOSYSTEM",
```

Comment:
Please increase the indent on lines 880-884 by 2 more spaces to be consistent with other examples.